src/__experimental__/components/switch/switch.stories.js

```diff
@@ -120,11 +119,7 @@ const validationKnobs = (type, storyName) => {
     name: `switch-${type}`,
     value: type,
     onChange: ev => handleChange(ev, type),
-    validations: testValidation('valid'),
-    warnings: testValidation('warn'),
-    info: testValidation('info'),
-    unblockValidation: true,
-    useValidationIcon: true
+    [type]: 'Message'
```

Comment:
@mkrds This doesn't work when its a boolean.

|Storybook boolean|
|--|
![image](https://user-images.githubusercontent.com/2328042/80135346-05613000-8598-11ea-8faf-6b3119b0da58.png)

|Storybook string|
|--|
![image](https://user-images.githubusercontent.com/2328042/80135309-f5495080-8597-11ea-9771-abab32148adb.png)

@harpalsingh please can you check this behaviour, the other inputs apply an outline to the input but the switch does not. It's not in the DLS.